.github/workflows/windows-build.yml

```diff
@@ -0,0 +1,52 @@
+name: Windows-build
+
+on: [push]
+
+jobs:
+  windows_cuda:
+    name: cuda102/release/shared
+    runs-on: [windows-latest]
+    steps:
+    - uses: actions/checkout@v2
+    - name: setup
+      run: |
+        choco install cuda -y
+    - name: configure
+      run: |
+        $env:ChocolateyInstall = Convert-Path "$((Get-Command choco).Path)\..\.."   
+        Import-Module "$env:ChocolateyInstall\helpers\chocolateyProfile.psm1"
+        refreshenv
+        mkdir build
+        cd build
+        $env:PATH="$pwd\windows_shared_library;$env:PATH"
+        cmake -DGINKGO_BUILD_CUDA=ON -DGINKGO_BUILD_OMP=OFF ..
```

Comment:
There is no way to test OpenMP on Windows?

Reply:
I add mingw/cygwin to test OpenMP